src/pages/UserPage/UserPage.tsx

```diff
@@ -49,18 +151,46 @@ const UserPage: FC = () => {
     })
   }
 
+  const handleDeactivateModal = (title: string) => {
+    showModal(ModalTypes.UserAndRoleManagement, {
+      title: title,
+      cancelButtonContent: t('button.cancel'),
+      modalContent: t('modal.deactivate_user_content'),
+      className: classes.deactivateContent,
+      handleProceed: handleSubmit(onSubmit),
+      dynamicForm: <DynamicForm fields={fields} control={control} />,
+    })
+  }
+
+  const currentFormattedDate = dayjs(new Date()).format('YYYY-MM-DD')
```

Comment:
**[non-bocker]**

dayjs() does the same thing as dayjs(new Date()), so the "new Date()" part is unnecessary with dayjs